src/components/checkboxFilter/index.tsx

```diff
@@ -18,34 +18,55 @@ function CheckboxFilter<ItemKey extends string, FilterName extends string>({
   const hasGroups = items.some(
     (item) => (item.childCheckboxes ?? []).length > 0,
   );
+
+  const itemsPerColumn = Math.ceil(items.length / numberOfColumnsOnDesktop);
+
+  // Create an array to store items grouped by columns
+  const columns = Array.from(
+    { length: numberOfColumnsOnDesktop },
+    (_, columnIndex) => {
+      const columnStartIndex = columnIndex * itemsPerColumn;
+      const columnEndIndex = Math.min(
+        (columnIndex + 1) * itemsPerColumn,
+        items.length,
+      );
+      return [...items].slice(columnStartIndex, columnEndIndex);
+    },
+  );
+
   return (
     <TranslationProvider language={language} scopes={['checkboxFilter']}>
       <Box
         sx={{
```

Comment:
`sx` is only supposed to be used if you have custom css that chakra does not support. Here you can add the `Grid` component